.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
Is the intention to only use Node 20 for development or also for building + production? 
If so, we need to update dockerfiles as they are using node16.

Reply:
Yes, all! https://github.com/lightdash/lightdash/issues/7030 contains the list we should follow to achieve the closure of this ticket